c/tskit/core.c

```diff
@@ -526,9 +526,13 @@ tsk_strerror_internal(int err)
                   "(TSK_ERR_BAD_SAMPLE_PAIR_TIMES)";
             break;
         case TSK_ERR_BAD_TIME_WINDOWS:
-            ret = "Time windows must be strictly increasing and end at infinity. "
+            ret = "Time windows must start at zero and be strictly increasing. "
                   "(TSK_ERR_BAD_TIME_WINDOWS)";
             break;
+        case TSK_ERR_BAD_TIME_WINDOWS_END:
```

Comment:
we needed to disambiguate the various notions of "bad" as different stats have different requirements now